src/main/js/main_html5.js

```diff
@@ -138,7 +138,7 @@ require("../../../html5-common/js/utils/environment.js");
     this.maxSupportedElements = (function() {
       var iosRequireSingleElement = OO.isIos;
       var androidRequireSingleElement = OO.isAndroid &&
-                                        (!Platform.isAndroid4Plus || OO.chromeMajorVersion < 40);
+                                        (!OO.isAndroid4Plus || OO.chromeMajorVersion < 40);
```

Comment:
Yes - i had originally kept a local copy because the one in html5-common wasn't correct.  It was updated a long time ago but this was never deprecated